headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,94 @@
+#ifndef HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+#define HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+
+struct LiveActor {
+    uint16_t id;     /* The id of the Actor in the actor list */
+    uint16_t kind;   /* The specie of the Actor */
+    bool is_enabled; /* true when the actor is loaded, false otherwise (should be checked if unsure)
+                      */
+    undefined field3_0x5;
+    uint16_t hanger;
+    uint8_t sector;
+    int8_t field6_0x9;
+    int16_t field7_0xa;
+    struct vec2_uint collision_box; /* The size of the collision box of the pokemon */
+    undefined field9_0x14;
+    undefined field10_0x15;
+    undefined field11_0x16;
+    undefined field12_0x17;
+    struct vec2_uint size_div2; /* The size of the collision box divided by two */
+    undefined field14_0x20;
+    undefined field15_0x21;
+    undefined field16_0x22;
+    undefined field17_0x23;
+    undefined field18_0x24;
+    undefined field19_0x25;
+    undefined field20_0x26;
+    undefined field21_0x27;
+    struct vec2_int limit_min_pos; /* minimum possible coordinates, for random move in free roam */
+    struct vec2_int limit_max_pos; /* maximum possible coordinates, for random move in free roam */
+    undefined maybe_command_data[236]; /* Seems to be a script-related struct */
+    int16_t field25_0x124;
+    undefined field26_0x126;
+    undefined field27_0x127;
+    uint32_t bitfied_collision_layer; /* not sure */
+    int32_t field29_0x12c;
+    uint8_t current_direction; /* not sure */
+    undefined field31_0x131;
+    undefined field32_0x132;
+    undefined field33_0x133;
+    undefined field34_0x134;
+    undefined field35_0x135;
+    undefined field36_0x136;
+    undefined field37_0x137;
+    struct vec2_int field38_0x138;
+    undefined field39_0x140;
+    undefined field40_0x141;
+    undefined field41_0x142;
+    undefined field42_0x143;
+    undefined field43_0x144;
+    undefined field44_0x145;
+    undefined field45_0x146;
+    undefined field46_0x147;
+    undefined field47_0x148;
+    undefined field48_0x149;
+    undefined field49_0x14a;
+    undefined field50_0x14b;
+    uint32_t field51_0x14c;
+    int16_t field52_0x150;
+    bool field53_0x152;
+    undefined field54_0x153;
+    int16_t field55_0x154;
+    bool field56_0x156;
+    undefined field57_0x157;
+    int16_t field58_0x158;
+    int8_t direction;
+    undefined field60_0x15b;
+    struct vec2_int coord_min; /* the top-left coordinate of the collision box of the actor */
+    struct vec2_int coord_max; /* the bottom-right coordinate of the collision box of the actor */
+    undefined4 field63_0x16c;
+    undefined4 field64_0x170;
+    int8_t field65_0x174;
+    int8_t field66_0x175;
+    int16_t field67_0x176;
+    int16_t field68_0x178;
+    undefined field69_0x17a;
+    undefined field70_0x17b;
+    int32_t movement_related; /* not sure. seem to be 0 when not moving and 3 when moving most of
```

Comment:
For comments that don't fit on one line, I usually like to put them on the line above.